docs/Reference/TypeScript.md

```diff
@@ -661,6 +661,22 @@ However, there are a couple of suggestions to help improve this experience:
 - Make sure the `no-unused-vars` rule is enabled in
   [ESLint](https://eslint.org/docs/rules/no-unused-vars) and any imported plugin
   are actually being loaded.
+- In case you've the `@typescript-eslint/no-floating-promises` enabled,
+please double-check that your `.eslintrc` includes `allowForKnownSafePromises`
```

Comment:
Now that [ESLint's new flat config](https://eslint.org/blog/2022/08/new-config-system-part-2) is the default, `.eslintrc` is no longer what many repos will use instead of `eslint.config.*`:

```suggestion
please double-check that your ESLint configuration includes a `allowForKnownSafePromises`
```